prow/cmd/tide/README.md

```diff
@@ -1,180 +1,30 @@
 # Tide
 
 Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
-component for managing a pool of PRs that match a given set of criteria.
-It will automatically retest and merge PRs in the pool if they pass tests
-against the latest base branch commit.
+component for managing a pool of GitHub PRs that match a given set of criteria.
+It will automatically retest and merge PRs in the pool if they have passing and
+up-to-date test results.
 
-Configuration of Tide is located under
-[prow/config.yaml](https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml)
-file. It consists of two parts:
+[Open Issues](https://github.com/kubernetes/test-infra/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3Aarea%2Ftide)
 
-  * **Specification of PR pools/criteria**:
-    A set of all PRs Tide keeps track of.
-    The PRs are periodically checked if they are ready to be merged.
-    Configured under `tide` key of the configuration file.
+## Documentation
+- [I JUST WANT MY PR TO MERGE!](/prow/cmd/tide/pr-authors.md)
+- [Configuring Tide](/prow/cmd/tide/config.md)
+- [Maintainer's Guide](/prow/cmd/tide/maintainers.md)
 
-  * **Specification of PR tests**:
-    A set of jobs that are run over each PR before it gets merged.
-    No job is required to be provided in which case it's enough if a PR has all
-    relevant labels set. If a set of jobs is specified,
-    all must succeed (unless a job is configured with `skip_report: true`).
-    Configured under `presubmits` key of the configuration file.
 
-## PR pools
+## Features
+- Automatic batch testing and merging of multiple PRs.
+- Ensures that PRs are tested against the most recent base branch commit before they are allowed to merge.
+- GitHub status context that indicates if each PR is in a pool or what requirements are missing.
+- Block merge to individual branches or whole repos using specifically labelled GitHub issues.
+- Prometheus metrics.
+- Support for 'optional' status contexts that aren't required for merge.
+- Serves live data about current actions and pools which can be consumed by [Deck](/prow/cmd/deck) to populate the Tide dashboard and the PR dashboard.
+- A single instance can provide merge automation to dozens of orgs and repos with unique merge criteria.
```

Comment:
call out merges to branches are disjoint

Reply:
Done.